.github/workflows/ci.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
           with:
             python-version: '3.x'
         - name: Install
-          shell: bash -l {0}
+          shell: bash -e -l {0}
```

Comment:
```suggestion
```
Or, put, differently, what's the advantage to having this? (Sorry, we may have had this discussion. I just don't remember.)

Reply:
I dont remember either. For some reason, we must have thought of using a login shell here.

Reply:
Kill it?